package.json

```diff
@@ -21,17 +21,21 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-builder-config#readme",
   "dependencies": {
+    "@blackbaud/skyux-logger": "1.0.1",
```

Comment:
This library has been updated since you first submitted your PR, so we probably wanna pull that version in.